src/Tasks/NETMessage.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Tasks.Deployment.Bootstrapper;
```

Comment:
nit: I think it's better to declare the enum here (or just do if/switch on the string) than reference it from what looks like unrelated code.